src/rsz/src/RepairSetup.cc

```diff
@@ -53,8 +53,16 @@
 #include "sta/Sdc.hh"
 #include "sta/TimingArc.hh"
 #include "sta/Units.hh"
+#include "sta/VerilogWriter.hh"
 #include "utl/Logger.h"
 
+/*void
+writeVerilog(const char *filename,
+             bool sort,
+             bool include_pwr_gnd,
+             sta::CellSeq *remove_cells,
+             sta::Network *network);
+*/
```

Comment:
rm

Reply:
yep, sorry some old debug code (was looking at verilog at various points in flow).